docs/migrations/v4-to-v5.md

```diff
@@ -0,0 +1,390 @@
+# Migrating from v4 to v5
+
+This document is intended to highlight significant changes which may require more information than presented in the CHANGELOG.
+Any changes that must be done by a user of ibc-go should be documented here.
+
+There are four sections based on the four potential user groups of this document:
+- Chains
+- IBC Apps
+- Relayers
+- IBC Light Clients
+
+**Note:** ibc-go supports golang semantic versioning and therefore all imports must be updated to bump the version number on major releases.
+```go
+github.com/cosmos/ibc-go/v4 -> github.com/cosmos/ibc-go/v5
+```
+
+## Chains
+
+### Ante decorator
+
+The `AnteDecorator` type in `core/ante` has been renamed to `RedundantRelayDecorator` (and the corresponding constructor function to `NewRedundantRelayDecorator`). Therefore in the function that creates the instance of the `sdk.AnteHandler` type (e.g. `NewAnteHandler`) the change would be like this:
+
+```diff
+func NewAnteHandler(options HandlerOptions) (sdk.AnteHandler, error) {
+   // parameter validation
+
+   anteDecorators := []sdk.AnteDecorator{
+      // other ante decorators
+-     ibcante.NewAnteDecorator(opts.IBCkeeper),
++     ibcante.NewRedundantRelayDecorator(options.IBCKeeper),
+   }
+
+   return sdk.ChainAnteDecorators(anteDecorators...), nil
+}
+```
```

Comment:
Can we link to the changes in the pr that changed this to show how it changed in our simapp app.go?

Reply:
This actually changed twice, since we renamed the ante decorator twice... So I link to the [PR](https://github.com/cosmos/ibc-go/pull/1820) that did the last renaming?

Reply:
yes, sounds good to me! You can maybe comment on that section of the code in the pr that the name was changed twice

Reply:
Just realized that no changes were required actually in `app.go`, so I just added a link to the PR.